examples/python-guide/logistic_regression.py

```diff
@@ -1,9 +1,7 @@
 # coding: utf-8
 """Comparison of `binary` and `xentropy` objectives.
-
```

Comment:
please revert all of these unrelated whitespace changes

Reply:
I see that you've marked this "resolved", but these changes are still in the PR. You can see them on the "Files Changed" tab, at https://github.com/microsoft/LightGBM/pull/4356/files.

![image](https://user-images.githubusercontent.com/7608904/121236724-dec58d80-c85b-11eb-8ebd-e327b099c40f.png)

Please add blank lines back to these places where they were removed.

Reply:
I don't know why that happened. I just copied the whole code and modified those 4 lines with f-string formation.

Reply:
You may be using an editor that automatically removes such lines or something like that. Thanks for removing them.